lib/server/services/qualityControlFlag/QcFlagService.js

```diff
@@ -314,6 +316,117 @@ class QcFlagService {
         return summary;
     }
 
+    /**
+     * Set list of DPL detectors which contribute to GAQ (global aggregated quality)
+     * for given runs and data pass
+     *
+     * @param {object} scope of definition
+     * @param {number} scope.dataPassId data pass id
+     * @param {number[]} scope.runNumbers list of unique run numbers
+     * @param {number[]} scope.dplDetectorIds list of unique DPL detector ids
+     * @return {Promise<void>} promise
+     */
+    setGaqDetectors({ dataPassId, runNumbers, dplDetectorIds }) {
```

Comment:
Why not passing actual parameters ? Scope make sense when you have varying parameters, I think it's not the case here. 
Also please try to be more descriptive in the parameters description

Reply:
refactored